systemd/CMakeLists.txt

```diff
@@ -1,5 +1,26 @@
 set(CMAKE_INSTALL_SYSTEMD_SERVICEDIR "${CMAKE_INSTALL_LIBEXECDIR}/systemd/system"
   CACHE PATH "Location for systemd service files")
+set(CEPH_SYSTEMD_ENV_DIR "/etc/sysconf"
+  CACHE PATH "Location for systemd service environmental variable settings files")
```

Comment:
with this change, the RPM spec file will need to run cmake with `-DCEPH_SYSTEMD_ENV_DIR=/etc/sysconfig`?

Reply:
no need to do so. as the default value is "/etc/sysconf".